source/chapters/vinyl_control.rst

```diff
@@ -148,6 +150,13 @@ At the present time, Serato records are recommended if you are looking to buy
 vinyl. If you want to use CDs, you can download a free copy for personal use
 from `Serato`_.
 
+Mixxx has initial support for the Traktor MK2 timecodes. Why initial? This
+timecode is very different from the others, which makes processing it more
+demanding in terms of complexity, CPU load and memory requirements. The
+support is also restricted to signals amplified with a phono-preamp that
+applies the RIAA De-Emphasis properly. Simply amplifying the phono signal in
+software will not work.
```

Comment:
Are there any specific issues users should be aware of?  Like, what are the "dangers", if any, of using this timecode

Reply:
it sounds like high CPU use is one

Reply:
I updated the descriptions. CPU load is actually not much higher. A little bit, but not that much that it should be mentioned. The memory consumption can increase up to 100MB for the CD. Should I mention those numbers here?